test/vcf/optional_policy_test.cpp

```diff
@@ -135,6 +135,51 @@ namespace ebi
                                 vcf::InfoBodyError*);
             }
         }
+
+        SECTION("SVLEN test")
+        {
+                source->meta_entries.emplace(vcf::INFO,
```

Comment:
There seems to be a silly issue with the indentation. I think you used 2 tabs instead of 1 (after the beginning of the SECTION block) in line 41 and below (till 181). Make it 1 tab.